corehq/apps/ota/views.py

```diff
@@ -64,7 +64,11 @@ def search(request, domain):
         search_es = search_es.case_property_query(key, value, fuzzy=(key in fuzzies))
     results = search_es.values()
     if should_use_sql_backend(domain):
-        cases = [get_instance_from_data(CommCareCaseSQLSerializer, sqlify_result(result)) for result in results]
+        # This is always going to fail validation, because case_id is marked as unique, and the instance will
+        # always already exist:
+        # cases = [get_instance_from_data(CommCareCaseSQLSerializer, sqlify_result(result)) for result in results]
+        # Ugly DB hit, mitigated only by CASE_SEARCH_MAX_RESULTS:
+        cases = CommCareCaseSQL.objects.filter(case_id__in=[result['_id'] for result in results])
```

Comment:
Pretty sure we can get around this.  Either way you can't use Django managers with sharded models. You'd have to do this

CaseAccessors (domain).get_cases (case_ids)

Excuse phone typing